explorer/app/components/Header/DrawerContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from "@mui/material";
+import React from "react";
+import { LogoProps } from "../Logo/Logo";
+import { NavLinksProps, NavLinks } from "../NavLinks/NavLinks";
+import { SocialLinksProps, SocialLinks } from "../SocialLinks/SocialLinks";
+import {
+  DrawerContainer,
+  SloganDrawerContainer,
+  SocialDrawerContainer,
+} from "./Header.styles";
+
+interface DrawerContentProps {
+  logo: LogoProps;
+  navLinks: NavLinksProps;
+  socialLinks: SocialLinksProps;
+}
+
+export const DrawerContent = ({
+  logo,
+  navLinks,
+  socialLinks,
+}: DrawerContentProps) => {
```

Comment:
For consistency with other projects, let's add the return statement `JSX.Element` here and and throughout 🙏🏽

Reply:
oh right! My bad, I've forgotten to fix this on some components 🤦‍♂️